rules/cross-platform/privilege_escalation_sudo_buffer_overflow.toml

```diff
@@ -0,0 +1,87 @@
+[metadata]
+creation_date = "2021/02/03"
+maturity = "production"
+updated_date = "2021/02/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies the attempted use of a Heap-based Buffer Overflow vulnerability for the Sudo binary in Unix-like
+systems (CVE-2021-3156). Successful exploitation allows an unprivileged user to escalate to the root user.
+"""
+false_positives = [
+    """
+    This rule could generate false positives if the process arguments leveraged by the expoit are shared by custom
+    scripts using the Sudo or Sudoedit binaries.
+    """,
+]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Possible Sudo Heap-based Buffer Overflow Vulnerability Attempt (CVE-2021-3156)"
```

Comment:
maybe something like "Potential Privilege Escalation via Sudoedit Exploit"

Reply:
i dont think you need the word `Possible` and `Attempt`. 
Could do without one, i.e. -  `Sudo Heap-based Buffer Overflow Vulnerability Attempt (CVE-2021-3156)`

Reply:
Yeah, this title is a mess.

What about this, from CISA?
> Sudo Heap-Based Buffer Overflow Vulnerability (CVE-2021-3156)

Reply:
I think even the below will make sense. The CVE is mentioned in the description.
> Sudo Heap-Based Buffer Overflow Attempt

OR

> Attempted Sudo Heap-Based Buffer Overflow

Reply:
Disco. Going with it.